docs/contributors/repository-management.md

```diff
@@ -116,6 +116,8 @@ Every pull request goes through a manual code review, in addition to automated t
 
 *As a contributor*, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
 
+Code reviews are encouraged by everyone who is willing to attempt one. If you review a pull request and are confident in the changes, approve it. If you don't feel totally confident it is ready for merging, add your review with a comment that says it should have another set of eyes on it before final approval. This can help filter out obvious bugs and simplify reviews for core members. It also is a great learning experience as you can follow the later reviews to see how to improve your own future reviews.
+
```

Comment:
Might be good to add that it's also fine to just comment with questions when you're unsure of how something should work or if the reason for the change is unclear. Or review only the parts of the code you are comfortable with, and comment that changes x and y haven't been looked at yet. 
Great to emphasise code review as a learning experience 🙂

Reply:
Thanks for the feedback! I took a stab at what you were suggesting, let me know what you think.

Reply:
Looks great, thanks for this!